tests/test_from_nir.py

```diff
@@ -17,7 +17,7 @@ def _torch_model_map(m: nir.NIRNode, device: str = "cpu") -> torch.nn.Module:
         lin.weight.data = torch.nn.Parameter(torch.tensor(m.weight).to(device))
         return lin
     elif isinstance(m, nir.Input) or isinstance(m, nir.Output):
-        return None
+        return torch.nn.Identity()
```

Comment:
was this change necessary? I had some problems tracing tensors through `torch.nn.Identity` nodes, so I'm afraid adding these will make the conversion of the resulting `GraphExecutor` module back to NIR fail (because the input and output tensor is the same for `torch.nn.Identity` nodes, it messes up the extracted graph's connectivity)

Reply:
No problem to keep it if it's necessary, we should anyway fix the underlying problem to allow tracing of Identity nodes

Reply:
The issue with not having a node without a module is that the graph execution fails without introducing further checks and hacks. And the second problem is that the graph then doesn't have a clear set of input and output nodes. I am unclear for now of what a clean solution would be.

Would it be possible to execute `ignore_node` method to ignore identity nodes?

Reply:
@Jegp any concerns with this? otherwise I'd be okay to merge this

Reply:
This is a test, no? So it shouldn't have much impact.